jsonschema-core.xml

```diff
@@ -2198,6 +2199,33 @@ Content-Type: application/json;
                     </artwork>
                 </figure>
 
+                <t>
+                    Media type parameters are also used in HTTP's Accept request header:
+                </t>
+
+                <figure>
+                    <artwork>
+<![CDATA[
+Accept: application/json;
+          schema="http://example.com/qiang http://example.com/li",
+        application/json;
+          schema="http://example.com/kumar"
+]]>
+                    </artwork>
+                </figure>
+
+                <t>
+                    As with Content-Type, multiple schema parameters in the same string
+                    requests an instance that conforms to all of the listed schemas.
+                </t>
+
+                <t>
+                    The Accept header also allows repeating the same media type with
+                    different parameters, which requests that the instance conforms
+                    to one parameter value or the other.  Such an instance can conform
+                    to all separately specified parameters but is not guaranteed to do so.
+                </t>
```

Comment:
I think I misread this the first time through, I thought this was saying a media type could have a property specified multiple times like `application/json;schema="http://example.com/a";schema="http://example.com/b"` but I don't know what that would even mean. I suspect it's invalid.

This is all technically correct, so I don't have an objection as such, but I'd be clearer to me _without_ this paragraph. I think the example above is sufficient to get the point across.

I can't find if there's a good way to distinguish the "type slash subtype" media type, versus the media type of a document (including its media type parameters).

Reply:
@awwright thanks, I'll take a look at the wording.  I think I was trying to cover a question from the issue that was filed (this came from an outside request filed a number of months ago), but maybe it can be trimmed or reworded.

Reply:
@awwright thanks for catching this, it really was confusing when I went back and read it.  I think I have clarified it by just highlighting the example.  Please let me know if this works for you.

Reply:
@handrews thanks for making these changes - one minor suggested tweak.  Instead of "conforms to at least one of the identified schemas", it might be clearer to say "conforms to both the first and second schema or conforms to the third".